include/tvm/relay/type.h

```diff
@@ -93,6 +93,15 @@ class TensorTypeNode : public BaseTensorTypeNode {
 
 RELAY_DEFINE_NODE_REF(TensorType, TensorTypeNode, Type);
 
+/*! \brief possible kinds of TypeVar */
+enum Kind : int {
```

Comment:
If we want to move it out, then it should not be called Kind, but instead TypeVarKind

Reply:
In this particular case, I do think keep the Kind field inside the class makes a lot of sense